### ISSUE TITLE
about the training data

### ISSUE THREAD

**Issue by viko-3** [ml-jku/cloome#6]
Thanks for your good job, but I have a question about data.
In your paper, 
<img width="1247" alt="image" src="https://github.com/ml-jku/cloome/assets/55910336/0970c525-f3eb-4904-8875-24d67fb212ab">

there are 674,357 training data paris, but I download the data from https://github.com/gigascience/paper-bray2017, in the datasplits.tar.gz, there are 3 datasplit-train.csv(datasplit1-train.csv,datasplit2-train.csv,datasplit3-train.csv)，the all pairs are about 417,317
<img width="1016" alt="image" src="https://github.com/ml-jku/cloome/assets/55910336/86ad1160-cf88-4746-b581-1f01823e828c">

So, how can I get the training data with 674,357 pairs?
-----------------------------------------------------------------------
sorry, the the all pairs are less than 600,000, how can I get the training data with 674,357 pairs?

**Comment by anasf97** [comment]
Hi, 

Thank you for reaching out. The splits in datasplits.tar.gz, correspond to the split in Hofmarcher et. al, where they used images corresponding to bioactivity-labeled compounds. In our case, we used the same splits concerning molecules, but with the difference that we also included every other molecule that was not present in the validation or test set, as in this case, we didn't need labels to train the model.

The full dataset can be found  originally here (http://gigadb.org/dataset/100200) and the training split with the 674,357  pairs can be found here (https://huggingface.co/anasanchezf/cloome/).

I hope this helps!

Best, 
Ana

**Comment by anasf97** [comment]
Hi,

Thank you for reaching out. The splits in datasplits.tar.gz, correspond to the split in Hofmarcher et. al, where they used images corresponding to bioactivity-labeled compounds. In our case, we used the same splits concerning molecules, but with the difference that we also included every other molecule that was not present in the validation or test set, as in this case, we didn't need labels to train the model.

The full dataset can be found originally here (http://gigadb.org/dataset/100200) and the training split with the 674,357 pairs can be found here (https://huggingface.co/anasanchezf/cloome/).

I hope this helps!

Best,
Ana
